assets/js/googlesitekit-api.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import API from 'assets/js/googlesitekit/api';
+
+if ( typeof global.googlesitekit === 'undefined' ) {
+	throw new Error( '`googlesitekit` is undefined. You need to import `googlesitekit` to use the `googlesitekit-api` library.' );
+}
+
+global.googlesitekit.api = API;
```

Comment:
Can we use `window` here instead, like elsewhere?

Reply:
We can, but in the spirit of #920 and this being a refactored set of code I figured I'd go with best practices. This gets converted to `window` by Webpack anyway, so I figured it was better to start using it here. There's really no reason to use `window` over `global` though.

Reply:
(For now I've used `window` though, for consistency. 😄)